hack/python-sdk/post_gen.py

```diff
@@ -26,7 +26,9 @@
     ("import kubeflow.training", "from kubeflow.training.models import *"),
     ("kubeflow.training.models.v1\/.*.v1.", "V1"),
     ("kubeflow.training.models.kubeflow/org/v1/", "kubeflow_org_v1_"),
+    ("kubeflow.training.models.runtime/raw_extension.runtime\.", "Runtime"),
```

Comment:
Where do we use it ?

Reply:
And why do we have runtime model in our APIs ?

Reply:
I don't think it's used in the APIs, but it's being added by the OpenAPI specification generation script here:
https://github.com/kubernetes/code-generator/blob/b15df6411b47bf6e80bfc63947af6b436b2e05c6/kube_codegen.sh#L365-L367

It's hard-coded and it doesn't seem like there is an easy way to get rid of it. That being said, I don't think it actually impacts anything.